pylib/Tools/Build/Autotools.py

```diff
@@ -197,6 +199,16 @@ def execute(self, log, keyvals, testDef):
                     # this is a multistep operation, and so we need to
                     # retain the output from each step in the log
                     log['autogen'] = (stdout, stderr)
+
+                    # Use mpi4py to get MPI information for IUDatabase plugin
+                    log['mpi_name'] = MPI.get_vendor()[0]
```

Comment:
That is true. If the MPI version is swapped out or re-installed, then mpi4py would have to be re-installed.

I will work on an implementation using "mpirun --version" and show it to Noah when he's available.